helm/cluster-openstack/templates/openstack_cluster.yaml

```diff
@@ -53,11 +54,7 @@ spec:
             name: {{ .Values.subnetName }} 
       {{- end }}
       {{- if .Values.bastion.bootFromVolume }}
-      image: ""
       rootVolume:
-        sourceType: image
         diskSize: {{ .Values.bastion.diskSize }}
-        sourceUUID: {{ .Values.bastion.image | quote }}
-      {{- else }}
-      image: {{ .Values.bastion.image | quote }}
       {{- end }}
+      image: {{ .Values.bastion.image | quote }}
```

Comment:
@bavarianbidi do we need to change something in kubectl-gs for this change?

Reply:
As with CAPO `v1alpha5` the two fields for refering the image are now called `image` and `imageUUID` (https://cluster-api-openstack.sigs.k8s.io/topics/crd-changes/v1alpha4-to-v1alpha5.html#changes-to-rootvolume) we can still use the existing input parameter for the image in `kubectl-gs` is e.g. `bastion-image` but have to use the `image name` instead of the `uuid` here.

After this PR got merged and we have a new version for `cluster-openstack` we only have to bump the default version for this in `kubectl-gs`.